playground/src/results.tsx

```diff
@@ -53,6 +53,17 @@ function resultIsSame(a: ShotResult, b: ShotResult): boolean {
   return true;
 }
 
+function MatrixLatex(props: { latex: string }) {
+  return (
+    <>
+      <div style="margin: 12px 4px; font-size: 1em; font-style: italic;">
+        DumpOperation
```

Comment:
Does this need the title here? I know this is just for the playground, but I think it would be fine to remove this, as messages and dump machine/register output don't have a title.